test/specs/controller.bubble.tests.js

```diff
@@ -273,160 +273,152 @@ describe('Bubble controller tests', function() {
 		expect(meta.data[4] instanceof Chart.elements.Point).toBe(true);
 	});
 
-	it('should set hover styles', function() {
-		var chart = window.acquireChart({
-			type: 'bubble',
-			data: {
-				datasets: [{
-					data: [{
-						x: 10,
-						y: 10,
-						r: 5
-					}, {
-						x: -15,
-						y: -10,
-						r: 1
-					}, {
-						x: 0,
-						y: -9,
-						r: 2
-					}, {
-						x: -4,
-						y: 10,
-						r: 1
+	describe('Interactions', function() {
+		function triggerElementEvent(chart, type, el) {
```

Comment:
can probably move this out into a helper once this is merged. I think similar code is used in a number of tests